sdk/maps/azure-maps-search/tests/recordings/test_client_e2e.test_fuzzy_search_top.yaml

```diff
@@ -10,13 +10,22 @@ interactions:
       - keep-alive
       User-Agent:
       - azsdk-python-maps-search/1.0-preview Python/3.9.13 (Windows-10-10.0.22000-SP0)
+      subscription-key:
+      - <RealSubscriptionKey>
       x-ms-client-id:
       - <RealClientId>
     method: GET
-    uri: https://atlas.microsoft.com/search/fuzzy/json?api-version=1.0&query=Taiwan%20High%20Speed%20Rail&limit=5&subscription-key=pryWmPOH5EtRpxLMlGoMFni8b5rgPnMSR3ZFtrDj0hY
```

Comment:
Please revoke this subscription key. This key should not be available anymore.

Reply:
Its removed.